crates/blockchain/mempool.rs

```diff
@@ -103,6 +103,19 @@ pub fn remove_transaction(hash: &H256, store: &Store) -> Result<(), StoreError>
     store.remove_transaction_from_pool(hash)
 }
 
+pub fn get_nonce(address: &Address, store: &Store) -> Result<Option<u64>, MempoolError> {
+    let pending_filter = PendingTxFilter {
+        ..Default::default()
+    };
+
+    let pending_txs = filter_transactions(&pending_filter, store)?;
+    let empty_vec = vec![];
```

Comment:
Nit:
Here I think it would be better to avoid the vec allocation and do:
```rust
let nonce =  match pending_txs.get(address) { 
   Some(txs) => txs.last().map(|tx| tx.nonce() + 1),
   None => None
}
```

Reply:
Totally agree!

Reply:
Done!